tests/functional/test_confirm_email_address.py

```diff
@@ -1,59 +1,55 @@
 import pytest
 from flask import url_for
 
-email_depts = [('some.one@digital.cabinet-office.gov.uk',
-                'Government Digital Service')]
-
 
+@pytest.mark.usefixtures('live_server')
 class When_on_confirm_email_with_yes_selected_and_valid_email(object):
 
-    @pytest.mark.parametrize("email_address,department", email_depts)
     def it_goes_to_department_confirm_when_continue_clicked(
-            self, live_server, browser, email_address, department):
-        browser.visit(url_for('main.request_email_address', _external=True))
+            self, browser, submit_known_email_address):
```

Comment:
Just a suggestion, feel free to ignore.

Since this class sets up the context of the test as "When on confirm email (page), with yes selected and valid email (submitted)", you could use a `setup_method`, eg:
```python
@pytest.mark.usefixtures('live_server')
class When_on_confirm_email_with_yes_selected_and_valid_email(object):
    email_address = 'some.one@digital.cabinet-office.gov.uk'
    department = 'Government Digital Service'

    def setup_method(self):
        set_email_known(True)
        set_email_address(self.email_address)

    def it_goes_to_department_confirm_when_continue_clicked(self, browser):
        browser.find_by_css('form button').click()
        assert browser.url == url_for('main.confirm_dept', _external=True)
        ...
```
This would make this test class pretty close to a BDD style given/when structure